lib/samples/create_load_report/create_load_report.dart

```diff
@@ -0,0 +1,446 @@
+    setState(() => _ready = false);
+    try {
+      // Create a service geodatabase from the Naperville Electric feature service.
+      final serviceGeodatabase = ServiceGeodatabase.withUri(
+        Uri.parse(
+          'https://sampleserver7.arcgisonline.com/server/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer',
+        ),
+      );
+      await serviceGeodatabase.load();
+      // Initialize the utility network with the service geodatabase.
+      _utilityNetwork = UtilityNetwork(serviceGeodatabase);
+      await _utilityNetwork.load();
+
+      // Get the network source, asset group, and asset type for the starting location.
+      final networkSource = _utilityNetwork.definition?.networkSources
+          .firstWhere((ns) => ns.name == 'Electric Distribution Device');
+      final assetGroup = networkSource?.assetGroups.firstWhere(
+        (ag) => ag.name == 'Circuit Breaker',
+      );
+      final assetType = assetGroup?.assetTypes.firstWhere(
+        (at) => at.name == 'Three Phase',
+      );
+      final terminal = assetType?.terminalConfiguration?.terminals.firstWhere(
+        (t) => t.name == 'Load',
+      );
+      final globalId = Guid.fromString(
+        '{1CAF7740-0BF4-4113-8DB2-654E18800028}',
+      );
+
+      // Create the default starting location.
+      _startingLocation = _utilityNetwork.createElementWithAssetType(
+        assetType!,
+        globalId: globalId!,
+        terminal: terminal,
+      );
+
+      // Get the utility tier.
+      _utilityTier = _utilityNetwork.definition?.domainNetworks
+          .firstWhere((definition) => definition.name == 'ElectricDistribution')
+          .tiers
+          .firstWhere((tier) => tier.name == 'Medium Voltage Radial');
+
+      // Get the default trace configuration as the base condition.
+      _baseCondition =
+          _utilityTier!
+                  .getDefaultTraceConfiguration()!
+                  .traversability!
+                  .barriers!
+              as UtilityTraceConditionalExpression;
+
+      // Create downstream trace parameters with function outputs.
+      _traceParameters = UtilityTraceParameters(
+        UtilityTraceType.downstream,
+        startingLocations: [_startingLocation],
+      );
+      // Specify the result types to include elements and function outputs.
+      _traceParameters.resultTypes.addAll([
+        UtilityTraceResultType.elements,
+        UtilityTraceResultType.functionOutputs,
+      ]);
+      // Assign the trace configuration to trace parameters.
+      _traceParameters.traceConfiguration = _createTraceConfiguration();
+
+      // Create a list of possible phases from a given network attribute
+      _phaseCodedValues = _createPhaseList();
+
+      // Set the ready state variable to true to enable the sample UI.
+      setState(() {
+        _phaseDataList = getPhaseDataList();
+        _ready = true;
+      });
+    } on Exception catch (e) {
+      if (!mounted) return;
```

Comment:
I don't think we need to check `!mounted`. That's already part of our `setState()` override.